google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITBucketSnippets.java

```diff
@@ -291,6 +294,69 @@ public void testDisableLifecycleManagement() {
     assertEquals(0, storage.get(BUCKET).getLifecycleRules().size());
   }
 
+  @Test
+  public void testGetPublicAccessPrevention() {
+    storage
+        .get(BUCKET)
+        .toBuilder()
+        .setIamConfiguration(
+            BucketInfo.IamConfiguration.newBuilder()
+                .setPublicAccessPrevention(BucketInfo.PublicAccessPrevention.ENFORCED)
+                .build())
+        .build()
+        .update();
+    PrintStream standardOut = System.out;
+    final ByteArrayOutputStream snippetOutputCapture = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(snippetOutputCapture));
+    GetPublicAccessPrevention.getPublicAccessPrevention(PROJECT_ID, BUCKET);
+    String snippetOutput = snippetOutputCapture.toString();
+    System.setOut(standardOut);
+    assertTrue(snippetOutput.contains("enforced"));
+    storage
+        .get(BUCKET)
+        .toBuilder()
+        .setIamConfiguration(
+            BucketInfo.IamConfiguration.newBuilder()
+                .setPublicAccessPrevention(BucketInfo.PublicAccessPrevention.UNSPECIFIED)
+                .build())
+        .build()
+        .update();
+  }
+
+  @Test
+  public void testSetPublicAccessPreventionEnforced() {
+    SetPublicAccessPreventionEnforced.setPublicAccessPreventionEnforced(PROJECT_ID, BUCKET);
```

Comment:
Can we call out here why we don't need to first set the bucket to `UNSPECIFIED` before this line? (In the test case for unspecified it does set enforced before setting `UNSPACIFIED`)